test/tap/tests/test_ps_hg_routing-t.cpp

```diff
@@ -60,6 +60,19 @@ int main(int argc, char** argv) {
 	MYSQL_QUERY(mysqladmin, "insert into mysql_query_rules (rule_id, active, flagIN, match_digest, negate_match_pattern, re_modifiers, destination_hostgroup, apply) values (200, 1, 0, '^SELECT', 0, 'CASELESS', 1, 1)");
 	MYSQL_QUERY(mysqladmin, "load mysql query rules to runtime");
 
+	for (;;) {
```

Comment:
This means wait forever?

Reply:
I forgot to add a safety counter -- the infinite loop should be ended after 60 seconds. This is issue is fixed with last commit.